`coq_makefile`'s `findlib_remove` does not work on Windows

<!-- Thank you for reporting a bug to Coq! -->

#### Description of the problem

https://github.com/mit-plv/fiat-crypto/runs/5149557822?check_suite_focus=true
```
# INSTALL BigZ/ZMake.glob D:\a\fiat-crypto\fiat-crypto\_opam\lib/coq//user-contrib/Bignums\\BigZ
# INSTALL BigZ/BigZ.glob D:\a\fiat-crypto\fiat-crypto\_opam\lib/coq//user-contrib/Bignums\\BigZ
# # findlib needs the package to not be installed, so we remove it before
# # installing it
# /bin/sh: D:afiat-cryptofiat-crypto_opambin/ocamlfind.exe: No such file or directory
# make[1]: *** [Makefile.coq:601: install] Error 127
# make[1]: Leaving directory '/cygdrive/d/a/fiat-crypto/fiat-crypto/_opam/.opam-switch/build/coq-bignums.dev'
# make: *** [Makefile:14: install] Error 2
```

There's a quoting issue, I'll prepare a PR to fix it

#### Coq Version

master

--- a/tools/CoqMakefile.in
+++ b/tools/CoqMakefile.in
@@ -175,14 +175,14 @@ findlib_install = \
 	$(HIDE)if [ "$(METAFILE)" ]; then \
 	  $(FINDLIBPREINST) && \
 	  mv $(METAFILE) $(METAFILE).skip ; \
-	  $(OCAMLFIND) install $(2) $(FINDLIBDESTDIR) $(FINDLIBPACKAGE) $(1); \
+	  "$(OCAMLFIND)" install $(2) $(FINDLIBDESTDIR) $(FINDLIBPACKAGE) $(1); \
 	  rc=$$?; \
 	  mv $(METAFILE).skip $(METAFILE); \
 	  exit $$rc; \
 	fi
 findlib_remove = \
 	$(HIDE)if [ ! -z "$(METAFILE)" ]; then\
-	  $(OCAMLFIND) remove $(FINDLIBDESTDIR) $(FINDLIBPACKAGE); \
+	  "$(OCAMLFIND)" remove $(FINDLIBDESTDIR) $(FINDLIBPACKAGE); \
 	fi
 
 # Substitution of the path by appending $(DESTDIR) if needed.

